src/components/cc-addon-credentials-beta/cc-addon-credentials-beta.smart-keycloak.js

```diff
@@ -80,7 +80,7 @@ defineSmartComponent({
     });
     updateComponent('docLink', {
       text: i18n('cc-addon-credentials-beta.doc-link.keycloak'),
-      href: generateDocsHref('/doc/addons/keycloak/'),
```

Comment:
@HeleneAmouzou note that this was a mistake from me 🙈 the docs paths given to `generateDocsHref` should never start with `/doc/` because the helper already handles that part, check that you didn't copy this mistake in other products if you can (or we'll check in review, not a big deal).